diff -ru xml-schemas/portfoliomanager-schemas-26.0/propertyUse/convenienceStoreWithGasStation.xsd xml-schemas/portfoliomanager-schemas-27.0/propertyUse/convenienceStoreWithGasStation.xsd
--- xml-schemas/portfoliomanager-schemas-26.0/propertyUse/convenienceStoreWithGasStation.xsd	2026-07-06 23:30:56.192384256 +0000
+++ xml-schemas/portfoliomanager-schemas-27.0/propertyUse/convenienceStoreWithGasStation.xsd	2026-07-07 06:33:05.618219894 +0000
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="26.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="27.0">
     <xs:include schemaLocation="characteristics.xsd"/>
     <xs:include schemaLocation="propertyUse.xsd"/>
     <xs:element name="convenienceStoreWithGasStation" type="convenienceStoreWithGasStationType">
@@ -28,6 +28,9 @@
                         <xs:element ref="numberOfFTEWorkers" minOccurs="0"/>
                         <xs:element ref="numberOfCookingEquipmentUnits" minOccurs="0"/>
                         <xs:element ref="numberOfWarmingHeatingEquipmentUnits" minOccurs="0"/>
+                        <xs:element ref="grossFloorAreaUsedForFoodPreparation" minOccurs="0"/>
+                        <xs:element ref="singleStore" minOccurs="0"/>
+                        <xs:element ref="exteriorEntranceToThePublic" minOccurs="0"/>
                     </xs:all>
                 </xs:complexType>
             </xs:element>
diff -ru xml-schemas/portfoliomanager-schemas-26.0/propertyUse/convenienceStoreWithoutGasStation.xsd xml-schemas/portfoliomanager-schemas-27.0/propertyUse/convenienceStoreWithoutGasStation.xsd
--- xml-schemas/portfoliomanager-schemas-26.0/propertyUse/convenienceStoreWithoutGasStation.xsd	2026-07-06 23:30:56.192384256 +0000
+++ xml-schemas/portfoliomanager-schemas-27.0/propertyUse/convenienceStoreWithoutGasStation.xsd	2026-07-07 06:33:05.618235154 +0000
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="26.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="27.0">
     <xs:include schemaLocation="characteristics.xsd"/>
     <xs:include schemaLocation="propertyUse.xsd"/>
     <xs:element name="convenienceStoreWithoutGasStation" type="convenienceStoreWithoutGasStationType">
@@ -28,6 +28,9 @@
                         <xs:element ref="numberOfFTEWorkers" minOccurs="0"/>
                         <xs:element ref="numberOfCookingEquipmentUnits" minOccurs="0"/>
                         <xs:element ref="numberOfWarmingHeatingEquipmentUnits" minOccurs="0"/>
+                        <xs:element ref="grossFloorAreaUsedForFoodPreparation" minOccurs="0"/>
+                        <xs:element ref="singleStore" minOccurs="0"/>
+                        <xs:element ref="exteriorEntranceToThePublic" minOccurs="0"/>
                     </xs:all>
                 </xs:complexType>
             </xs:element>
diff -ru xml-schemas/portfoliomanager-schemas-26.0/propertyUse/foodSales.xsd xml-schemas/portfoliomanager-schemas-27.0/propertyUse/foodSales.xsd
--- xml-schemas/portfoliomanager-schemas-26.0/propertyUse/foodSales.xsd	2026-07-06 23:30:56.193566536 +0000
+++ xml-schemas/portfoliomanager-schemas-27.0/propertyUse/foodSales.xsd	2026-07-07 06:33:05.618421140 +0000
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="26.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="27.0">
     <xs:include schemaLocation="characteristics.xsd"/>
     <xs:include schemaLocation="propertyUse.xsd"/>
     <xs:element name="foodSales" type="foodSalesType">
@@ -25,6 +25,9 @@
                         <xs:element ref="percentHeated" minOccurs="0"/>
                         <xs:element ref="areaOfAllWalkInRefrigerationUnits" minOccurs="0"/>
                         <xs:element ref="lengthOfAllOpenClosedRefrigerationUnits" minOccurs="0"/>
+                        <xs:element ref="grossFloorAreaUsedForFoodPreparation" minOccurs="0"/>
+                        <xs:element ref="singleStore" minOccurs="0"/>
+                        <xs:element ref="exteriorEntranceToThePublic" minOccurs="0"/>
                     </xs:all>
                 </xs:complexType>
             </xs:element>
diff -ru xml-schemas/portfoliomanager-schemas-26.0/propertyUse/retail.xsd xml-schemas/portfoliomanager-schemas-27.0/propertyUse/retail.xsd
--- xml-schemas/portfoliomanager-schemas-26.0/propertyUse/retail.xsd	2026-07-06 23:30:56.197566536 +0000
+++ xml-schemas/portfoliomanager-schemas-27.0/propertyUse/retail.xsd	2026-07-07 06:33:05.618909805 +0000
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="26.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="27.0">
 	<xs:include schemaLocation="characteristics.xsd"/>
 	<xs:include schemaLocation="propertyUse.xsd"/>
 	<xs:element name="retail" type="retailType">
@@ -27,6 +27,7 @@
                         <xs:element ref="areaOfAllWalkInRefrigerationUnits" minOccurs="0"/>
                         <xs:element ref="lengthOfAllOpenClosedRefrigerationUnits" minOccurs="0"/>
                         <xs:element ref="cookingFacilities" minOccurs="0"/>
+						<xs:element ref="grossFloorAreaUsedForFoodPreparation" minOccurs="0"/>
 					</xs:all>
 				</xs:complexType>
 			</xs:element>
diff -ru xml-schemas/portfoliomanager-schemas-26.0/propertyUse/supermarket.xsd xml-schemas/portfoliomanager-schemas-27.0/propertyUse/supermarket.xsd
--- xml-schemas/portfoliomanager-schemas-26.0/propertyUse/supermarket.xsd	2026-07-06 23:30:56.201566536 +0000
+++ xml-schemas/portfoliomanager-schemas-27.0/propertyUse/supermarket.xsd	2026-07-07 06:33:05.619000457 +0000
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="26.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="27.0">
 	<xs:include schemaLocation="characteristics.xsd"/>
 	<xs:include schemaLocation="propertyUse.xsd"/>
 	<xs:element name="supermarket" type="supermarketType">
@@ -25,6 +25,9 @@
 						<xs:element ref="percentHeated" minOccurs="0"/>
                         <xs:element ref="areaOfAllWalkInRefrigerationUnits" minOccurs="0"/>
                         <xs:element ref="lengthOfAllOpenClosedRefrigerationUnits" minOccurs="0"/>
+						<xs:element ref="grossFloorAreaUsedForFoodPreparation" minOccurs="0"/>
+						<xs:element ref="singleStore" minOccurs="0"/>
+						<xs:element ref="exteriorEntranceToThePublic" minOccurs="0"/>
 					</xs:all>
 				</xs:complexType>
 			</xs:element>
diff -ru xml-schemas/portfoliomanager-schemas-26.0/propertyUse/wholesaleClubSupercenter.xsd xml-schemas/portfoliomanager-schemas-27.0/propertyUse/wholesaleClubSupercenter.xsd
--- xml-schemas/portfoliomanager-schemas-26.0/propertyUse/wholesaleClubSupercenter.xsd	2026-07-06 23:30:56.201566536 +0000
+++ xml-schemas/portfoliomanager-schemas-27.0/propertyUse/wholesaleClubSupercenter.xsd	2026-07-07 06:33:05.619122678 +0000
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="26.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="27.0">
 	<xs:include schemaLocation="characteristics.xsd"/>
 	<xs:include schemaLocation="propertyUse.xsd"/>
 	<xs:element name="wholesaleClubSupercenter" type="wholesaleClubSupercenterType">
@@ -27,6 +27,7 @@
                         <xs:element ref="areaOfAllWalkInRefrigerationUnits" minOccurs="0"/>
                         <xs:element ref="lengthOfAllOpenClosedRefrigerationUnits" minOccurs="0"/>
                         <xs:element ref="cookingFacilities" minOccurs="0"/>
+						<xs:element ref="grossFloorAreaUsedForFoodPreparation" minOccurs="0"/>
 					</xs:all>
 				</xs:complexType>
 			</xs:element>
